### PR TITLE
Create instruction for integrating PingOne for customer SAML solution

### PR DESCRIPTION
**Pull request by brycewang03**

### CODE REVIEW

File: HowTos/SAML_Integration_PingOne_IdP.rst [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+  |pingone_idp_retrieve_idp_metadata_url|
+
+.. _pingone_update_saml_endpoint:
+
+Step 4. Update Aviatrix SP Endpoint
+###################################
+
+.. note::   
+   This step is usually completed by the Aviatrix admin. PineOne IdP provides IdP Metadata through URL obtained in Retrieve `PingOne IdP metadata URL <#pingone-idp-metadata>`__ step. PingOne for Customers IdP requires a custom SAML request template.
```
Comment (emereeclaire) [inline HowTos/SAML_Integration_PingOne_IdP.rst:176]: Is the  "custom SAML request template." the one provided below in (.. code-block:: XML) line 202-209.
Is it the same one used for both controller and OpenVPN : Custom SAML Request Template?

File: HowTos/SAML_Integration_PingOne_IdP.rst [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+      <samlp:NameIDPolicy xmlns:samlp="urn:oasis:names:tc:SAML:2.0:protocol" Format="urn:oasis:names:tc:SAML:2.0:nameid-format:transient" SPNameQualifier="$SPNameQualifier" AllowCreate="true"></samlp:NameIDPolicy>
+      <samlp:RequestedAuthnContext xmlns:samlp="urn:oasis:names:tc:SAML:2.0:protocol" Comparison="exact"><saml:AuthnContextClassRef xmlns:saml="urn:oasis:names:tc:SAML:2.0:assertion">urn:oasis:names:tc:SAML:2.0:ac:classes:PasswordProtectedTransport</saml:AuthnContextClassRef>
+      </samlp:RequestedAuthnContext>
+   </samlp:AuthnRequest>
+   
+.. _pingone_test_integration:
+
+Step 5. Test the Integration
+#############################
+
```
Comment (emereeclaire) [inline HowTos/SAML_Integration_PingOne_IdP.rst:215]: In https://docs.aviatrix.com/HowTos/SAML_Integration_Okta_IdP.html, There is a tip (that tells the user to assign users to the new application).

Is it intentional here that we don't mention this step anymore?

File: HowTos/SAML_Integration_PingOne_IdP.rst [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+   ||pingone_idp_configuring_attribute_mapping||
+
+#. Click Save and Close.
+
+#. Enable the WEB SAML APP
+
+   |pingone_idp_enable|
+   
+.. _pingone_idp_metadata:
+
```
Comment (emereeclaire) [inline HowTos/SAML_Integration_PingOne_IdP.rst:152]: In reference to https://docs.aviatrix.com/HowTos/SAML_Integration_Okta_IdP.html -- in the last step, it indicates that the application needs to be assigned to an account. 

Is there a need to do that here to complete (Step 2. Create a PingOne Web SAML App for Aviatrix)?

File: HowTos/SAML_Integration_PingOne_IdP.rst (hunk 1 above)
Comment (brycewang03) [reply HowTos/SAML_Integration_PingOne_IdP.rst:176]: Yes, that is the same one used for both controller and OpenVPN

File: HowTos/SAML_Integration_PingOne_IdP.rst (hunk 2 above)
Comment (brycewang03) [reply HowTos/SAML_Integration_PingOne_IdP.rst:215]: PingOne for customer does not have that concept

File: HowTos/SAML_Integration_PingOne_IdP.rst (hunk 3 above)
Comment (brycewang03) [reply HowTos/SAML_Integration_PingOne_IdP.rst:152]: PingOne for customer does not have that concept. Basically user and application are global as far as I know.

**Pull request merged**